DQM/DTMonitorClient/src/DTLocalTriggerLutTest.cc

```diff
@@ -271,19 +271,17 @@ void DTLocalTriggerLutTest::runClientDiagnostic(DQMStore::IBooker& ibooker, DQMS
           int phiNoData = 0;
           int phibNoData = 0;
           for (int stat = 1; stat <= 4; ++stat) {
-            switch (static_cast<int>(phiWhSummary->GetBinContent(sect, stat))) {
-              case 1:
-                phiNoData++;
-              case 2:
-              case 3:
-                phiErr++;
+            int res = static_cast<int>(phiWhSummary->GetBinContent(sect, stat));
+            if (res == 1) {
+              phiNoData++;
+            } else if (res == 3) {
+              phiErr++;
             }
```

Comment:
@mrodozov 
Umm, but isn't replacing the switch with if-else essentially equivalent to adding breaks? I mean, if `res` is
* 1: earlier the coded did `phiNoData++; phiErr++;`, and now only `phiNoData++;`
* 2: earlier `phiErr++;`, now nothing
* 3: earlier and now `phiErr++;`

(and no, I don't know which behavior is the correct one, if either)

Reply:
Hm, never written switch statements without breaks before, I was assuming it wouldn't go into a case "condition" if it doesn't match but it will. So now I don't know if that code was intentionally left without breaks, adding breaks will lead to the if else condition in the update indeed, better ask the authors again.